CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+
+## nf-core/rnafusion version 1.0dev - <date>
```

Comment:
version `1.0dev` or version `1.0`?

Reply:
Nice catch. Once approved I will make the change 👍